# HG changeset patch
# Parent  7661b6f47576f84d3b4f7d01a894691f81c06b8f
Exclude slides/*.pdf from sources tarballs during release

They just take some unnecessary space.

diff --git a/dist/release.py.in b/dist/release.py.in
--- a/dist/release.py.in
+++ b/dist/release.py.in
@@ -455,6 +455,7 @@
             # without the .tar extension):
             "-s", "/^\\./{}/".format(archive_name.rsplit(".", 1)[0]),
             "--exclude", ".hg",
+            "--exclude", "slides/*.pdf",
             "."
         ])
     os.makedirs(LIGHTSD_ARCHIVES_DIR, exist_ok=True)
